### DISCUSSION TITLE
Bank Value Item Toggles

### DISCUSSION THREAD

**Discussion by Crownfire36** [runelite/runelite#18264] [Ideas]
c

**Comment by Alexsuperfly** [comment]
put it in your inventory

**Reply by Crownfire36** [reply]
This would allow you to toggle more than 28 items.

**Reply by Felanbird** [reply]
Another workaround could just be to use the normal bank tabs and just sort the bank in that tab, instead of the "not in a tab" section, and put the items you don't want counted in a different tab.
